tests/pipelines/test_api.py

```diff
@@ -225,7 +231,8 @@ def test_library_object_serialization_deserialization():
         LibraryObject('egg', 'FiLe:/weird/case.ext'),
         LibraryObject('whl', 'file.ext'),
         LibraryObject('whl', 's3:/s3/path/file.ext'),
-        LibraryObject('jar', 'dbfs:/dbfs/path/file.ext')
+        LibraryObject('jar', 'dbfs:/dbfs/path/file.ext'),
+        LibraryObject('maven', {'coordinates': 'com.package.name'})
```

Comment:
Can we also add an exclusion for this test since it's a serialization test, and that thing's a list?